Assignment2.yaml

```diff
@@ -0,0 +1,402 @@
+  /People:
+    get:
+      tags:
+      - "People"
+      summary: "Find list of People"
+      description: "get all the people resources"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /People/{PeopleId}:
+    get:
+      tags:
+      - "People"
+      summary: "Find people by ID"
+      description: "get a specific people resource"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "PeopleId"
+        in: "path"
+        description: "ID of people to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /People/schema:
+    get:
+      tags:
+      - "People"
+      summary: "Return Schema"
+      description: "Returns a json Schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/people"
+  /starship:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find list of starships"
+      description: "Returns a List of starship"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      - name: "model"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /starship/{starshipId}:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find starship by ID"
+      description: "Returns a single starship"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "starshipId"
+        in: "path"
+        description: "ID of starship to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /starship/schema:
+    get:
+      tags:
+      - "Starship"
+      summary: "Find starship by ID"
+      description: "Returns a json schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/starship"
+  /species:
+    get:
+      tags:
+      - "Species"
+      summary: "Find list of species"
+      description: "get all the species resources"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+      responses:
```

Comment:
`responses` occurs 2 times, gives compilation error. Check before you do your submission.